### DISCUSSION TITLE
Option Serial in Section mcu must be specified

### DISCUSSION THREAD

**Discussion by gpena208777** [xblax/flashforge_ad5m_klipper_mod#80] [General]
On the 2nd day with this mod, I now have a screen that says 

Option Serial in Section mcu must be specified .
Im lost. Any help would be great.

Thanks all.

**Comment by consp** [comment]
Did you edit the `printer.base.cfg`?

It should start like this:
```
[mcu eboard]
serial: /dev/ttyS1
baud: 230400
restart_method: command

[mcu]
serial: /dev/ttyS5
baud: 230400
restart_method: command
```

My guess is the `/dev/ttyS5` is missing.

See https://github.com/xblax/flashforge_adm5_klipper_mod/blob/master/printer_configs/printer.base.cfg

**Comment by consp** [comment]
Marking as closed unless notified otherwise.
